qui/clipboard.py

```diff
@@ -26,6 +26,11 @@
 via Qubes RPC """
 # pylint: disable=invalid-name,wrong-import-position
 
+# Must be imported before creating threads
+from .tray.gross_gtk3_bug_workaround import (
```

Comment:
while I agree completely with the sentiment, could we get more polite module and function names here? The "gross" and "disgusting" can be left as an exercise to the reader to figure out themselves ;)